From: Mark Sercombe <sercombe.joel.mark@gmail.com>
Date: Thu, 13 Aug 2026 20:38:47 +0200
Subject: [PATCH] maple_tree: fix comment typo

Fix a spelling mistake n a code comment.  This is a comment only change
with no functional impact.

Link: https://lore.kernel.org/20260813183847.474357-1-sercombe.joel.mark@gmail.com
Signed-off-by: Mark Sercombe <sercombe.joel.mark@gmail.com>
Reviewed-by: Andrew Morton <akpm@linux-foundation.org>
Cc: Alice Ryhl <aliceryhl@google.com>
Cc: Andrew Ballance <andrewjballance@gmail.com>
Cc: Liam R. Howlett <liam@infradead.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 lib/maple_tree.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/maple_tree.c b/lib/maple_tree.c
index b48bc4064ad20..a0542b491bc23 100644
--- a/lib/maple_tree.c
+++ b/lib/maple_tree.c
@@ -3590,7 +3590,7 @@ static inline void mas_wr_prealloc_setup(struct ma_wr_state *wr_mas)
 
 /**
  * mas_prealloc_calc() - Calculate number of nodes needed for a
- * given store oepration
+ * given store operation
  * @wr_mas: The maple write state
  * @entry: The entry to store into the tree
  *
-- 
2.47.3

